src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Bundle\MessengerBundle\Command;
+
+use Akeneo\Tool\Bundle\MessengerBundle\Query\PurgeDoctrineQueueQuery;
+use Doctrine\DBAL\DBALException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PurgeMessengerCommand extends Command
+{
+    protected static $defaultName = 'akeneo:messenger:purge';
+
+    private PurgeDoctrineQueueQuery $purgeDoctrineQueue;
+
+    public function __construct(PurgeDoctrineQueueQuery $purgeDoctrineQueue)
+    {
+        parent::__construct();
+
+        $this->purgeDoctrineQueue = $purgeDoctrineQueue;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription(
+                'Purges the messenger SQL table in terms of the given retention time (default is 120 minutes)'
```

Comment:
```suggestion
                'Purges the messenger SQL table in terms of the given retention time (default is 7200 seconds)'
```
I think it's more readable to indicate the real value, in seconds.
We want to avoid that an user, who would want to reduce to retention time, like 60 minutes, could be encouraged to put `--retention-time 60`.